content/enterprise_influxdb/v1.9/administration/authentication_and_authorization.md

```diff
@@ -0,0 +1,500 @@
+Use the ``Bearer`` authorization scheme:
+
+```
+Authorization: Bearer <myToken>
+```
+{{% note %}}
+Only unexpired tokens will successfully authenticate.
+Be sure your token has not expired.
+{{% /note %}}
+
+###### Example query request with JWT authentication
+```bash
+curl -G "http://localhost:8086/query?db=demodb" \
+  --data-urlencode "q=SHOW DATABASES" \
+  --header "Authorization: Bearer <header>.<payload>.<signature>"
+```
+
+## Authenticate Telegraf requests to InfluxDB
+
+Authenticating [Telegraf](/{{< latest "telegraf" >}}/) requests to an InfluxDB instance with
+authentication enabled requires some additional steps.
+In the Telegraf configuration file (`/etc/telegraf/telegraf.conf`), uncomment
+and edit the `username` and `password` settings.
+
+```toml
+###############################################################################
+#                            OUTPUT PLUGINS                                   #
+###############################################################################
+
+# ...
+
+[[outputs.influxdb]]
+  # ...
+  username = "example-username" # Provide your username
+  password = "example-password" # Provide your password
+
+# ...
+```
+
+Restart Telegraf and you're all set!
+
+## Authorization
+
+Authorization is only enforced once you've [enabled authentication](#set-up-authentication).
+By default, authentication is disabled, all credentials are silently ignored, and all users have all privileges.
+
+### User types and privileges
+
+#### Admin users
+Admin users have `READ` and `WRITE` access to all databases and full access to the following administrative queries:
+
+##### Database management
+- `CREATE DATABASE`
+- `DROP DATABASE`  
+- `DROP SERIES`
+- `DROP MEASUREMENT`  
+- `CREATE RETENTION POLICY`
+- `ALTER RETENTION POLICY`
+- `DROP RETENTION POLICY`  
+- `CREATE CONTINUOUS QUERY`
+- `DROP CONTINUOUS QUERY`  
+
+For more information about these commands, see [Database management](/influxdb/v1.8/query_language/manage-database/) and
+[Continuous queries](/influxdb/v1.8/query_language/continuous_queries/).
+
+##### User management
+- Admin user management
+  - [`CREATE USER`](#user-management-commands)
+  - [`GRANT ALL PRIVILEGES`](#grant-administrative-privileges-to-an-existing-user)
+  - [`REVOKE ALL PRIVILEGES`](#revoke-administrative-privileges-from-an-admin-user)
+  - [`SHOW USERS`](#show-all-existing-users-and-their-admin-status)  
+- Non-admin user management:  
+  - [`CREATE USER`](#user-management-commands)
+  - [`GRANT [READ,WRITE,ALL]`](#grant-read-write-or-all-database-privileges-to-an-existing-user)
+  - [`REVOKE [READ,WRITE,ALL]`](#revoke-read-write-or-all-database-privileges-from-an-existing-user)
+- General user management:  
+  - [`SET PASSWORD`](#reset-a-users-password)
+  - [`DROP USER`](#drop-a-user)  
+
+See [below](#user-management-commands) for a complete discussion of the user management commands.
+
+#### Non-admin users
```

Comment:
This was for OSS - in Enterprise, users can have more fine-grained permissions. But I think that has to be set up with the API, rather than through the CLI

Reply:
Ah, so it's not a "complete discussion", right?

Reply:
Does it makes sense (and does it cover all bases) to replace the entire "User management" section here with a link to [Set up users with fine-grained authorization](https://docs.influxdata.com/enterprise_influxdb/v1.9/guides/fine-grained-authorization/)?
If so, let's do that, and then publish. If that doesn't cover all bases, let's create another issue to track this change specifically, and push out this PR, which addresses many broken links and starts folks on the auth path.